gradle.properties

```
@@ -19,3 +19,4 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+API_KEY = e534ffbc2d474446c8538d23b1c7605c
```

sao lại public API_KEY lên đây rồi 
```suggestion
API_KEY = YOUR_API_KEY
```